ENH No limit on max_features for SelectFromModel (#31939)
scikit-learn/scikit-learn: doc/whats_new/upcoming_changes/sklearn.feature_selection/31939.enhancement.rst

- :class:`feature_selection.SelectFromModel` now does not force `max_features` to be

>   less than the number of input features.

Reviewer: nitpick
```suggestion
  less than or equal to the number of input features.
```

> - :class:`feature_selection.SelectFromModel` now does not force max_features to be <= _num_features(X).
>   This was unnecessary and it is actually better not to have this limit for hyperparameter optimization.
>   By :user:`Thibault <ThibaultDECO>`

Reviewer 2: ```suggestion
- :class:`feature_selection.SelectFromModel` now does not force `max_features` to be
  less than the number of input features.
  By :user:`Thibault <ThibaultDECO>`
```
